services/wireguard/connection/connection.go

```diff
@@ -148,13 +149,16 @@ func (c *Connection) Start(ctx context.Context, options connection.ConnectOption
 		return errors.Wrap(err, "failed while waiting for a peer handshake")
 	}
 
-	dnsIPs, err := options.DNS.ResolveIPs(config.Consumer.DNSIPs)
-	if err != nil {
-		return errors.Wrap(err, "could not resolve DNS IPs")
-	}
-	config.Consumer.DNSIPs = dnsIPs[0]
-	if err := c.dnsManager.Set(c.opts.DNSScriptDir, conn.InterfaceName(), config.Consumer.DNSIPs); err != nil {
-		return errors.Wrap(err, "failed to configure DNS")
+	// TODO: Add DNS support for windows with https://github.com/mysteriumnetwork/node/issues/1840
+	if runtime.GOOS != "windows" {
```

Comment:
Runtime condition :/
Also dont get this change, we're enforcing `system` DNS in desktop, but `system` DNS configuration on interface will be skipped?

Reply:
This is just a quick temporary fix for windows to be able to connect. I'm solving https://github.com/mysteriumnetwork/node/issues/2337 issue which you registered. Our system dns for windows doesn't work, it fails.